src-example/services/api/index.test.js

```diff
@@ -42,6 +42,14 @@ describe('parseSettings', () => {
   it('has passed method', () => {
     expect(parseSettings({ method: 'post' }).method).toBe('post')
   })
+
+  it('merges headers', () => {
+    const otherSettings = { headers: { foo: 'bar' }}
```

Comment:
Thank you, @dennis-tra. Just one thing: the lint is complaining about the lack of space between the last brackets on this line.